views/update/review-update-statement.html

```diff
@@ -0,0 +1,291 @@
+{% extends "layout.html" %}
+{% import "includes/date-macros.html" as dateMacros %}
+
+{% block pageTitle %}
+  Review update statement before submitting - {{ SERVICE_NAME }} - GOV.UK
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+{% set isProtectedInformation %}
+  {% if appData.is_secure_register == 1 %}
+    Yes
+  {% else %}
+    No
+  {% endif %}
+{% endset %}
+
+{% set noChange %}
+  {% if appData.update.no_change == 1 %}
+    Yes
+  {% else %}
+    No
```

Comment:
According to the prototype 'no' would be 'No, I do not need to make changes'

Reply:
Fixed